test/e2e/service_bootstrap.py

```diff
@@ -61,13 +59,14 @@ def create_vpc() -> str:
     return vpc_id
 
 
-def create_subnet(vpc_id: str) -> str:
+def create_subnet(ec2, vpc_id: str, az: str, cidr=VPC_SUBNET_CIDR_BLOCK[0]) -> str:
```

Comment:
No need to have `ec2` as a parameter if you're overriding on L64

Reply:
this was a miss - client passed as argument was supposed to be the only one